### PR TITLE
fix: warnings, again

### PR DESCRIPTION
**Pull request by gerard-queralt**

### CODE REVIEW

**Review by markmeliaa — APPROVED** [review]
This is the remix

File: Game/Scripts/UIComboManager.h [hunk 1]
```diff
@@ -56,7 +56,7 @@ class UIComboManager : public Script
 
 	float clearComboTimer;
 	bool clearCombo;
-	bool  isEffectEnabled;
+	int  isEffectEnabled;
```
Comment (MarcGonzalezMoratona) [inline Game/Scripts/UIComboManager.h:59]:
```suggestion
	int isEffectEnabled;
```

**Comment by markmeliaa** [comment]
Conflicts 👀

**Review by markmeliaa — DISMISSED** [review]

**Review by PaablooCH — APPROVED** [review]
all good 👍

**Review by markmeliaa — APPROVED** [review]

**Pull request merged**